build.gradle

```diff
@@ -52,3 +52,104 @@ task replicateSources << {
 		}
 	}
 }
+
+configurations {
+	bnd
+}
+
+dependencies {
+	bnd "biz.aQute.bnd:bnd:2.4.0"
+}
+
+repositories {
+	mavenLocal()
+
+	mavenCentral()
+}
+
+task jarBndTools(type: GradleBuild) {
```

Comment:
Lets not do it this way since this can't handle maven.  To do maven we will have to exec a system process (mvn clean package) so we might as well do it the same way for all 4 builds.  

Actually for now, lets keep the diff task separate from the build tasks just to simplify.  We will probably reconnect them in the future.